pkg/kubelet/server/server.go

```diff
@@ -406,6 +413,11 @@ func (s *Server) InstallDefaultHandlers() {
 	checkers = append(checkers, s.extendedCheckers...)
 	healthz.InstallHandler(s.restfulCont, checkers...)
 
+	if utilfeature.DefaultFeatureGate.Enabled(zpagesfeatures.ComponentStatusz) {
+		s.addMetricsBucketMatcher("statusz")
+		statusz.Install(s.restfulCont, ComponentKubelet, statusz.NewRegistry())
```

Comment:
This is a read-only endpoint in line with the other RO handlers here, but given that we're deprecating the read-only port, I don't love expanding the set of endpoints we expose here. Can we install this under the debugging handlers instead?

Reply:
done